src/stan/io/reader.hpp

```diff
@@ -995,8 +995,10 @@ namespace stan {
        * Cholesky factor for a correlation matrix.
        */
       inline matrix_t cholesky_corr(size_t K) {
+        using stan::error_handling::check_cholesky_factor_corr;
         matrix_t y(matrix(K,K));
```

Comment:
Shouldn't this just be `y(K,K)` to avoid creating a matrix then copying it into the new matrix?  The compiler may be smart enough not to do too much work.  I also can't find `matrix` --- is it a typedef being brought in from elsewhere?